### PR TITLE
Export callback changes for EC3

### PR DESCRIPTION
**Pull request by tejassutar2024**
- Added template id in export callback for EC3
- Added missing localization strings in template

### CODE REVIEW

File: packages/itwin/ec3-widget/src/components/Templates.tsx [hunk 1]
```diff
@@ -98,15 +98,15 @@ export const Templates = ({ onClickCreate, onClickTemplateTitle, onExportResult
...
+              {EC3Widget.translate("addNewTemplate")}
             </Button>
           </>
         ) : (
           <div className="ec3w-templates-list">
             <div className="ec3w-toolbar" data-testid="ec3-templates">
               <div className="ec3w-toolbar-left">
-                <Button startIcon={<SvgAdd />} onClick={onClickCreate} styleType="high-visibility" title="New Template">
+                <Button startIcon={<SvgAdd />} onClick={onClickCreate} styleType="high-visibility" title={EC3Widget.translate("newTemplate")}>
                   New
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/src/components/Templates.tsx:110]: The "New" string should also be translated.

File: change/@itwin-ec3-widget-react-17a64684-49ac-4812-9bc9-4812e43ddccd.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (Guillar1) [inline change/@itwin-ec3-widget-react-17a64684-49ac-4812-9bc9-4812e43ddccd.json:1]: You should add another changelog entry for the second change.

File: packages/itwin/ec3-widget/src/components/Templates.tsx (hunk 1 above)
Comment (Guillar1) [reply packages/itwin/ec3-widget/src/components/Templates.tsx:110]: "Export", "Delete", there is still some remaining.

File: packages/itwin/ec3-widget/public/locales/en/EC3Widget.json [hunk 3]
```diff
@@ -33,6 +33,9 @@
     "noTemplateMsg": "It looks like you haven't added any templates yet. Click the button below to create your first template.",
     "addNewTemplate": "Add New Template",
     "newTemplate": "New Template",
-    "templateOptions": "Template Options"
+    "templateOptions": "Template Options",
+    "New": "New",
+    "Export": "Export",
+    "Delete": "Delete"
```
Comment (Guillar1) [inline packages/itwin/ec3-widget/public/locales/en/EC3Widget.json:39]: camelCase.

**Review by Guillar1 — APPROVED** [review]

**Pull request merged**